oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md

```diff
@@ -0,0 +1,305 @@
+   - ORDS API is created to get Customer and Orders
+   - OIC consumes the ORDS API to get a Customer and Orders
+   - OIC enriches the response and sends consolidated Customer and Orders to the client
+
+ The following diagram shows the interaction between the systems involved in this use case.
+    ![Integration Architecture](images/lab-integration-scenario.png)
+
+Estimated Time: 30 minutes
+
+### Objectives
+
+In this lab, you will:
+
+* Connect to ORDS API using OAuth Client Credentials grant
+* Understand how to enrich and transform the payload
+
+
+### Prerequisites
+
+This lab assumes you have:
+
+* All previous labs completed.
+
+
+## Task 1: Create the Integration flow
+
+1. In the left Navigation pane, click **Design** &gt; **Integrations**.
+
+2. On the *Integrations page*, click **Create**.
+
+3. On the *Integration Style* dialog, select and click on **Application driven orchestration**.
+
+4. In the *Create New Integration* dialog, enter the following information:
+
+    | **Element**          | **Value**          |       
+    | --- | ----------- |
+    |Name | Customer Order Composite Service |
+    |Description | This integration shows you how to connect to the REST API secured with OAuth using OPEN API 3.0 format |
+    {: title="Create Integration Flow"}
+
+    Accept all other default values.
+
+5. Click **Create**.
+   ![Create Integration Flow](images/create-integration-initial.png)
+
+6. Click on **Horizontal** to change the layout to Horizontal
+
+## Task 2: Define the REST Trigger and Response Interface
+
+1. Search for the *REST Interface* connection which was created in the previous Lab and click on it. This invokes the REST  Adapter Configuration Wizard.
+
+2. On the *Basic Info* page,
+     - for the *What do you want to call your endpoint?* element, enter **getCustomerOrders**
+     - for the *What does this endpoint do?* element, enter *This endpoint defines the REST interface to get customer orders.*
+     - Click **&gt; (Next step)**.
+
+3. In the *Resource Configuration** page,
+    - for the *What does this operation do?* element, enter **get orders for customers.**
+    - for the *What is the endpoint's relative resource URI?*, enter **/customerOrders**
+    - for the *What action do you want to perform on the endpoint?*, select **GET**
+    - Select *Add and review parameters for this endpoint* checkbox
+    - Select *Configure this endpoint to receive the response* checkbox
+    - Click **&gt; (Next step)**.
+
+4. From the *Request Parameters* page, in the *Specify Query Parameter section*, click **Add**. A new row appears.
+    - Enter **customer_id** in the *Name* column and select **string** from the *Date Type* column
+    - Click **&gt; (Next step)**.
+    ![Query Parameter Page](images/rest-adapter-query-parameter-page.png)
+
+5. On the **Response** Page
+    - Select the *response payload format* to **JSON Sample**
+    - Click the **&lt&lt&ltinline&gt&gt&gt** link.
```

Comment:
Is this right?

Reply:
Yes it is right the UX has <<inline>> so we need to espace